### PR TITLE
fix MPP-2298: remove the un-used /emails/ url

### PR DESCRIPTION
**Pull request by groovecoder**
This PR fixes MPP-2298.

How to test:

1. `pytest` should run with no errors nor failures
2. Create a random mask
   * [ ] It should still work
3. Create a domain mask
   * [ ] It should still work
4. Run the add-on (`web-ext run -s src/`)
5. Create a new mask with the add-on
   * [ ] It should still work

- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by say-yawn — COMMENTED** [review]
While testing that when the `emails_index` is enabled the `/emails` path still works I found out:
- when the waffle is created and switched on with an instance that is already running, you need to restart the server/instance for the switch to be used
- enabling `emails_index` switch and making a GET request on `/emails` results in the `Reverse for 'profile' not found. 'profile' is not a valid view function or pattern name.` error which was seen 7 days ago on prod ([sentry error](https://sentry.prod.mozaws.net/operations/fx-private-relay-prod/issues/19469691/?query=is%3Aunresolved%20transaction%3A%2Femails%2F))

All this to say I am supportive of us deleting the index view on the email app altogether. cc @jwhitlock

**Comment by groovecoder** [comment]
Updated to remove the url completely.

**Review by jwhitlock — APPROVED** [review]
Do you want to remove the backing code from `emails/views.py` as well, or do that later? If you do it all in one commit, it is easy to revert in the future if we decide we want it back.

If you don't, I _love_ deleting code, I can write the follow-up PR.

**Comment by groovecoder** [comment]
I removed it now. That should increase our code coverage too, right? 😄

File: emails/views.py [hunk 1]
```diff
@@ -47,7 +47,6 @@
     remove_trackers,
     count_all_trackers,
     get_message_content_from_s3,
-    get_post_data_from_request,
```
Comment (jwhitlock) [inline emails/views.py:50]: A few more imports can be removed now:

```
emails/views.py:23:1: F401 'django.contrib.messages' imported but unused
emails/views.py:25:1: F401 'django.core.exceptions.PermissionDenied' imported but unused
emails/views.py:27:1: F401 'django.http.JsonResponse' imported but unused
emails/views.py:28:1: F401 'django.shortcuts.redirect' imported but unused
emails/views.py:28:1: F401 'django.shortcuts.render' imported but unused
```

**Review by jwhitlock — CHANGES_REQUESTED** [review]
Yep, removing completely untested code should increase our code coverage 😁 

Thanks! Please remove a few more imports!

File: emails/views.py (hunk 1 above)
Comment (groovecoder) [reply emails/views.py:50]: removed!

**Review by jwhitlock — APPROVED** [review]
❤️ Thanks @groovecoder!

**Pull request merged**